### PR TITLE
[BathNES] Remove 'here' from post-report 'report a problem here' link

### PR DESCRIPTION
**Pull request by ludovic-tc**
Amend the top-left nav link on the report conf page to simply 'Report
a problem' rather than 'Report a problem here', with link directing
user back to main non-localised problem report page.

- [x] Whether this PR should include changes to any documentation, or the FAQ;
- [x] All cobrand-specific commits start their commit message with the cobrand in square brackets;
[skip changelog]

 Fixes mysociety/societyworks#2433

### CODE REVIEW

**Review by chrismytton — COMMENTED** [review]
Looking good so far! One comment about simplifying the tests.

If this is ready for a proper review then please do the following:

- Mark the PR as "Ready for review" (rather than draft)
- Tidy up commit history, i.e. remove "WIP" from commit messages and squash commits together where it makes sense.

**Comment by chrismytton** [comment]
@ludovic-tc I'll take a look at this tomorrow, but just to note if this is ready for review then the PR probably shouldn't be a draft. Also the "WIP" bit needs removing from the PR title, just to avoid confusion!

File: t/cobrand/bathnes.t [hunk 1]
```diff
@@ -220,7 +220,50 @@ subtest 'extra CSV columns are present if permission granted' => sub {
...
+                photo3        => '',
+                name          => $normaluser->name,
+                may_show_name => '1',
+                email         => $normaluser->email,
+                phone         => '',
+                category      => 'Other',
+            }
+        );
+
+        ok $mech->success, 'Successfully posted report';
```
[the remark this answers is not in the document]
Comment (ludovic-tc) [reply t/cobrand/bathnes.t]: added in fixup

**Review by dracos — APPROVED** [review]
Code change looks good, not sure it needed e.g. the attrs being customisable until a time when that was needed, but no harm I guess :) Please link PR descriptions to the ticket they're fixing, as per the help text when you're creating a new PR. It's https://github.com/mysociety/societyworks/issues/2433 in this case.

**Pull request merged**